src/bin/cargo-flamegraph.rs

```diff
@@ -120,44 +79,44 @@ fn build(opt: &Opt, kind: impl IntoIterator<Item = String>) -> anyhow::Result<Ve
         cmd.arg("--release");
     }
 
-    if let Some(ref package) = opt.package {
+    if let Some(package) = &opt.package {
```

Comment:
These changes seem unrelated (that is, unnecessary for porting to clap). Mind putting them in a separate PR or commit?